src/main/main.ts

```diff
@@ -394,6 +394,7 @@ const createWindow = async () => {
         : path.join(__dirname, '../../.erb/dll/preload.js'),
     },
   });
+  // mainWindow.webContents.openDevTools();
```

Comment:
Just a tip for the future - you can open up the dev tools using right click->inspect element or with the keyboard combo. No need any additional code. Unless you prefer to have it open all the time of course.

Reply:
Ah, I see. I guess I didn't realize you could do that with electron.